api/Element.json

```diff
@@ -1273,170 +2872,66 @@
             }
           },
           "status": {
-            "experimental": false,
+            "experimental": true,
```

Comment:
Again I don't understand why this is marked experimental (I know it is marked experimental in the MDN page, but that seems out of date).